README.rst

```diff
@@ -18,24 +18,35 @@ MyTardis
   :target: https://coveralls.io/r/mytardis/mytardis?branch=develop
   :alt: Coveralls Badge
 
-MyTardis is a multi-institutional collaborative venture that facilitates the
-archiving and sharing of data and metadata collected at major facilities such
-as the Australian Synchrotron and ANSTO and within Institutions.
+Overview
+--------
+MyTardis began at Monash University to solve the problem of users needing to
+store large datasets and share them with collaborators online. Its particular
+focus is on integration with scientific instruments, instrument facilities and
+research storage and computing infrastructure; to address the challenges of data
+storage, data access, collaboration and data publication.
+
+`Read more... <http://www.mytardis.org/about/>`_
+
+Key features for users
+----------------------
+The MyTardis data management platform is a software solution that manages research data and the associated metadata. MyTardis handles the underlying storage to ensure that data is securely archived and provides access to the data through a web portal. You can also access your data using an SFTP client like `Cyberduck <https://cyberduck.io/>`_.
```

Comment:
Small edits:
> The MyTardis data management platform is a software solution for managing research data and associated metadata. It provides researchers with a secure way to access, manage, share and publish their data through a simple web portal. MyTardis manages the underlying storage of data to ensure it is securely archived without unnecessary duplication.

Don't think we need to suggest a specific SFTP client on Github... most devs will have their favourite. How about:
> Data hosted in MyTardis can also be accessed via SFTP.